devdox/tests/unit_test/test_doubles/app/repository/api_key_repository_test_doubles.py

```
@@ -0,0 +1,49 @@
+from types import SimpleNamespace
+from typing import Any, List, Optional
+from uuid import UUID, uuid4
```

Ruff (0.11.9)

`uuid.UUID` imported but unused

Remove unused import: `uuid.UUID`

(F401)